src/Catty/ViewController/Download/Details/ProjectDetailStoreViewControllerExtension.swift

```diff
@@ -406,8 +404,11 @@ let kHTMLAHrefTagPattern = "href=\"(.*?)\""
         var mutableActiveLinkAttributes: [AnyHashable: Any] = [:]
         mutableActiveLinkAttributes[kCTForegroundColorAttributeName as String] = UIColor.brown
         mutableActiveLinkAttributes[kCTUnderlineStyleAttributeName as String] = NSNumber(value: false)
+    }
 
-        label.linkAttributes = mutableLinkAttributes
-        label.activeLinkAttributes = mutableActiveLinkAttributes
+    private func attributedLabel(_ label: ActiveLabel?, didSelectLinkWith url: URL?) {
```

Comment:
This function doesn't seem to do, what it is supposed to do.
Since you now use a ActiveLabel, you also need to handle it.
I would suggest having a look at the API calls for ActiveLabel. Especially at the *handleURLTap* :)